app/services/email_digest.rb

```diff
@@ -25,6 +25,7 @@ def send_periodic_digest_email
   private
 
   def get_users
+    # TODO: [@msarit] update query to call email_digest_periodic from correct table
```

Comment:
I need help reworking this query; `email_digest_periodic` now lives in `users_notification_setting`, not `users`. My attempts to call `.joins(:users_notification_settings)` on `User` did not work. 😞  I really need to strengthen my DB skills.

Reply:
@msartit You need to join on the association name you specified in `User`:

```ruby
User
  .registered
  .joins(:notification_setting)
  .where(notification_setting: { email_digest_periodic: true })
  .where.not(email: "")
```

Note that I specified explicitly that we want to use the `email_digest_periodic` column of the `users_notification_settings` table since it still exists on both tables.